shared/utility/support/yaml_expression.hpp

```diff
@@ -321,6 +321,52 @@ namespace YAML {
             return true;
         }
     };
+
+    template <int Rows, int Cols>
+    struct convert<Eigen::Matrix<double, Rows, Cols>> {
```

Comment:
Are these being used anywhere?

Reply:
encoding and decoding the ready position